app/partials/desktop/settings_modal.html

```diff
@@ -73,6 +73,11 @@
           <span class="tg_checkbox_label" my-i18n="settings_modal_message_preview"></span>
         </a>
 
+        <a class="md_modal_section_toggle_wrap tg_checkbox" ng-click="toggleDarkmode()" ng-class="theme.darkmode ? 'tg_checkbox_on' : ''">
```

Comment:
This would need to be changed to `ng-click="setTheme()" ng-class="theme.active_theme"`, and then `my-i18n="settings_modal_active_theme"`.

Instead of a boolean, we will need to make it accept user input, e.g., `dark`.